src/homework/mostOften/MostOften.java

```diff
@@ -0,0 +1,51 @@
+package homework.mostOften;
+
+import java.util.Arrays;
+
+public class MostOften {
+
+    public static void arrayToBollean(int[] myArr) {
+
+        boolean[] boolArr = new boolean[myArr.length - 1];
+
+        int count = 1;
+        int maxCount = 0;
+        int winner = 0;
+
+        for (int i = 0; i < myArr.length - 1; ++i) {
+            int[] subArray = Arrays.copyOfRange(myArr, i, i + 2);
+            boolArr[i] = subArray[0] <= subArray[1];
+//            System.out.print(Arrays.toString(subArray));
+//            System.out.println(boolArr[i]);
+        }
+
+//        System.out.println(Arrays.toString(boolArr));
+
+        for (int i = 0; i < boolArr.length - 1; ++i) {
+            int j = 1;
+            while (((i + j) < boolArr.length) && (boolArr[i] == boolArr[i + j])) {
```

Comment:
Generalnie staraj się unikać `while`, jeśli masz w tym miejscu `for`, a tu `for` aą się prosi